### ISSUE TITLE
Validation set?

### ISSUE THREAD

**Issue by mengxue-rs** [fudanxu/MSTAR-AConvNet#2]
When I read the paper, I can't find any descriptions about validation set which is used to select net structure & hyper parameters.
How many percents you split training set into validation set?
